docs/internationalization.md

```diff
@@ -0,0 +1,41 @@
+# Internationalization
+
+## Routing
+
+See [routing locales](routing.md#locale).
+
+## Messages
+
+The application uses [Symfony Translation](https://symfony.com/doc/current/translation.html) to manage translated messages 
+on the page (e.g. microcopy such as 'Displaying 100 results').
+
+Translations are stored in `translations/` and use the [ICU message format](https://symfony.com/doc/current/translation/message_format.html).
+
+It's important for messages to have
```

Comment:
@simonrjones End of sentence is missing

Reply:
Sorry! I added some more details on this, I think it makes sense to use nested keywords and to try to make these keywords as clear as possible. Hope this sounds OK.